R/write.R

```diff
@@ -14,7 +14,7 @@
 #' @keywords internal
 #'
 #' @examples
-#' \dontshow{.old_wd <- setwd(tempdir())}
+#' \dontshow{.old_wd <- setwd(tempdir()); proj_set(force = TRUE)}
```

Comment:
What is this in response to? Is it about not being able to create a nested project? I think, if we're starting to do this `.old_wd <- setwd(tempdir()`thing  in examples, we will have to be just as careful about these projects as we are with the temporary scooped project used in tests. See this:

https://github.com/r-lib/usethis/blob/01dbd8f40f35d08956a919e3d2680810da32deb6/tests/testthat/helper.R#L4-L17

Reply:
I don't know; it randomly started failing and I just hacked something to make it work

Reply:
Well, in any case, I have seen similar when session temp dir somehow has an `.Rproj` file in it. Which seems to happen periodically due to something RStudio does in the background but I've never been able to really get to the bottom of it.

I doubt this really has anything to do with this test and I feel like we may regret having a one-off `proj_set(force = TRUE)` in an example.

Reply:
If you see this again, the code above helps you figure out if you've somehow gotten an `.Rproj` file in session temp dir.